spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/logback/DefaultLogbackConfiguration.java

```diff
@@ -75,11 +76,13 @@ private void defaults(LogbackConfigurator config) {
 		String defaultCharset = Charset.defaultCharset().name();
 		config.getContext().putProperty("CONSOLE_LOG_CHARSET",
 				resolve(config, "${CONSOLE_LOG_CHARSET:-" + defaultCharset + "}"));
+		config.getContext().putProperty("CONSOLE_LOG_THRESHOLD", resolve(config, "${CONSOLE_LOG_THRESHOLD:-TRACE}"));
 		config.getContext().putProperty("FILE_LOG_PATTERN", resolve(config, "${FILE_LOG_PATTERN:-"
 				+ "%d{${LOG_DATEFORMAT_PATTERN:-yyyy-MM-dd'T'HH:mm:ss.SSSXXX}} ${LOG_LEVEL_PATTERN:-%5p} ${PID:- } --- [%t] "
 				+ "%-40.40logger{39} : %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}}"));
 		config.getContext().putProperty("FILE_LOG_CHARSET",
 				resolve(config, "${FILE_LOG_CHARSET:-" + defaultCharset + "}"));
+		config.getContext().putProperty("FILE_LOG_THRESHOLD", resolve(config, "${FILE_LOG_THRESHOLD:-TRACE}"));
```

Comment:
Isn't the default `DEBUG` rather than `TRACE`?

Reply:
It's filter level of appender, not log level of logger, TRACE means appender won't drop any log from logger by default.